### PR TITLE
feat(chromecast): Support Google Pixel Tablet

### PR DESCRIPTION
**Pull request by joeyparrish**
The new Pixel tablet behaves a little differently than older devices. This expands the tool to recognize the different home screen behavior and new status messages.

### CODE REVIEW

File: backends/chromecast/cast-utils.js [hunk 1]
```diff
@@ -116,18 +116,24 @@ function cast(flags, log, mode, url) {
...
-        if (data.type == 'RECEIVER_STATUS' && data.status.applications) {
-          appIds = data.status.applications.map((app) => app.appId);
+        let appLaunched = false;
+        let homeLaunched = false;
+        if (data.type == 'RECEIVER_STATUS') {
+          const applications = data.status.applications || [];
+          const appIds = applications.map((app) => app.appId);
+          appLaunched = appIds.includes(request.appId);
+          // On traditional Cast devices, there's a home screen app to run.
+          // On the new Google Pixel tablet (2013), the list becomes empty.
```
Comment (avelad) [inline backends/chromecast/cast-utils.js:126]: 2013? Shouldn't it be 2023?

File: backends/chromecast/cast-utils.js [hunk 2]
```diff
@@ -138,6 +144,12 @@ function cast(flags, log, mode, url) {
         } else if (data.type == 'RECEIVER_STATUS' && data.status.volume) {
           // Ignore updates on audio volume.  These occur frequently, are not
           // useful, and should not be logged below.
+        } else if (data.type == 'LAUNCH_STATUS' &&
+                   data.status == 'USER_PENDING_AUTHORIZATION') {
+          log.info('Waiting for user authorization to run the receiver...');
+        } else if (data.type == 'LAUNCH_STATUS' &&
```
Comment (alekzz) [inline backends/chromecast/cast-utils.js:150]: nit; I don't know in which cases LAUNCH_STATUS is received, but maybe worth add a `log.info()` in this scenario as well?

File: backends/chromecast/cast-utils.js (hunk 1 above)
Comment (joeyparrish) [reply backends/chromecast/cast-utils.js:126]: Ha!  Yes.  2023 for sure.

File: backends/chromecast/cast-utils.js (hunk 2 above)
Comment (joeyparrish) [reply backends/chromecast/cast-utils.js:150]: If the user chooses "always allow" in the cast permission dialog, then we see this status in subsequent launches.

**Review by alekzz — APPROVED** [review]

File: backends/chromecast/cast-utils.js (hunk 2 above)
Comment (joeyparrish) [reply backends/chromecast/cast-utils.js:150]: I've updated the comment to clarify the situation.  We also have logging for all unrecognized messages below, so I think we're covered.

File: backends/chromecast/cast-utils.js (hunk 1 above)
Comment (joeyparrish) [reply backends/chromecast/cast-utils.js:126]: Fixed!

**Pull request merged**